extensions/positron-python/package.json

```diff
@@ -753,9 +785,16 @@
                     "scope": "resource",
                     "type": "boolean",
                     "tags": [
-                        "experimental"
+                        "onExP",
+                        "preview"
                     ]
                 },
+                "python.REPL.provideVariables": {
```

Comment:
should we leave out/hide the native REPL settings as we expect people to use Positron's console?